client/js/tracking.js

```diff
@@ -11,9 +11,19 @@ export default {
       const componentName = component.getAttribute('data-component-name');
       const componentId = component.getAttribute('data-component-id');
       const clickedProperties = JSON.parse(clicked.getAttribute('data-track-click'));
-      const properties = Object.assign({}, clickedProperties, {componentId, action: 'click'});
-
-      trackEvent({name: componentName, properties});
+      const properties = Object.assign(
+        {},
+        clickedProperties,
+        {componentId, action: 'click'}
+      );
+      const ariaExpanded = clicked.getAttribute('aria-expanded');
+      const propertiesWithAria = ariaExpanded ? Object.assign(
+        {},
+        properties,
+        {ariaExpanded}
```

Comment:
I'm not sure we can, unless I'm missing something. At the time we render the page, we don't know about the expanded/collapsed state (which is added by JS from the show-hide component). Alternatively this logic could go in the show-hide component, but I don't think that makes any difference. Do you have other ideas?